contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,99 @@
+
+> Bad Hypothesis: Changing the color of the "Proceed to checkout" button will increase purchases.
+
+This is bad since it's unclear why we're testing this change and why we expect it to increase purchases. As a result, it's not clear what we need to measure in this experiment. Do we only need to count button clicks? Or is there something else we need to measure?
+
+Here's an improved version:
+
+> Good hypothesis: User research showed that users are unsure of how to proceed to the checkout page. Changing the button's color will lead to more users noticing it and thus more people will proceed to the checkout page. This will then lead to more purchases.
+
+It's now clear that we need to measure the following: 
+* Button clicks, to show if the change in color leads to more people noticing the button
+* Number of purchases, since more people arriving at the checkout page should mean more purchases
+
+This also makes it easier to investigate the experiment results, especially when they are not what we expected. For example:
+* If the color change didn't increase button clicks, there may be a different issue on the page.
+* If the number of button clicks increased but the number of purchases did not, there may be an issue with the checkout page.
+
+## 2. Not monitoring counter metrics
+
+Counter metrics measure unintended negative side-effects in your experiments. If you do not monitor them, you may accidentally roll out changes that result in a worse user experience. 
+
+For example, say you're testing a change in the number of marketing emails you send to your users. While the change may increase the number of visitors to your site, it may also increase the number of users who unsubscribe from your mailing list. In this case, the unsubscribe rate is a good counter metric to keep track of.
+
+## 3. Not testing experiment setup with a small rollout
+
+Sometimes we are so eager to get results from our experiments that we jump straight into testing it with all our users. This may be okay if everything is set up correctly, but if you've made a mistake, you may not be able to rerun your experiment. Let's look at why this is:
+
+Imagine you're an experiment with a 50/50 split between control and test. You roll out the experiment to all your users, but a day after launch you notice that your change is causing the app to crash for all test users. You immediately stop the experiment and fix the root cause of the crash. However, restarting the experiment now will produce unreliable results since many users have already seen your change.
+
+To avoid this problem, you should first test your experiment with a small rollout (e.g., 5% of users) for a few days. Once you're confident everything works correctly, you can start the experiment with the remaining users.
```

Comment:
You could mention that at PostHog, we always test in production ourselves and using our internal team before rolling out further.